src/routes/HomeScreen.tsx

```diff
@@ -118,23 +117,19 @@ const HomeScreen = ({ navigation, customDisciplines }: HomeScreenPropsType): JSX
   }
 
   return (
```

Comment:
I think using a SafeArea here is important here to adjust for different devices. For me the previous solution looks better even though it obvisouly has a bug somehwere